### DISCUSSION TITLE
OPNsense Firewall Nodes still saying Relayed

### DISCUSSION THREAD

**Discussion by AntGod6123** [dec0dOS/zero-ui#190] [General]
@dec0dOS First off thank you for this project, I'm loving the GUI and most things seem to be working well!

Is anyone else having issues with certain nodes continuing to say Relayed? I have added a OPNsense firewall node running Zerotier which runs on FreeBSD with the version number being 1.12.1. I have had previous times where the node will show up in the ZeroUI GUI and the controller (ubuntu server which also houses the ZeroUI Docker) connects all the nodes for access, but the system continues to say relayed. I'm curious what the script is doing to verify the node is Online as I have not found it yet? I have also enabled ZU_LAST_SEEN_FETCH which works well and I appreciate this feature!

Lastly, is there a chance that there will be an Idle Session Timeout implementation so when the system is inactive or restarted it forces the user to log back in?

**Comment by dec0dOS** [comment]
> Is anyone else having issues with certain nodes continuing to say Relayed?

You should probably investigate the problem inside the ZeroTier controller setup. ZeroUI is just a proxy API with the GUI. You could check ZeroTier FAQ and issues and discussions.

> Lastly, is there a chance that there will be an Idle Session Timeout implementation so when the system is inactive or restarted it forces the user to log back in?
 
Please open new feature request in order to track this feature.

**Reply by AntGod6123** [reply]
The fault was mine, I was lacking a rule to allow the controller to reach the firewall alongside the zerotier controllers. All is well!
